docs/api/internal.json

```diff
@@ -82,19 +82,38 @@
                             "example": false
                           },
                           "job_type":  {
```

Comment:
I think it should be something like this:

```
"job_type":  {
  "type": "array",
  "items" : {
    "type": "string",
    "enum": [
      "freelance",
      "full-time",
      "internship",
      "part-time",
      "temporary",
      "other"
    ]
  }
},
```

Reply:
@aaronfc, I was talking with @mikeyarce about this one. It seems there's a setting where the users can define multiple job types.
But JobTarget only accepts one. So since you're integrating the form, I'd suggest you consider receiving an array, but get only the first item to be used to pre-fill the job type field.

Reply:
Fixed in [513b102](https://github.com/Automattic/WP-Job-Manager/pull/2467/commits/513b1027ba481fe00f73f47d8ab300df43358408)